core/src/main/java/com/microsoft/applicationinsights/telemetry/ExceptionHandledAt.java

```diff
@@ -23,7 +23,10 @@
 
 /**
  * This enumeration is used by ExceptionTelemetry to identify if and where exception was handled.
+ *
+ * This Enum is now deprecated.
  */
+@Deprecated
```

Comment:
Was this replaced by something else? If so, let's add the `@deprecated` javadoc field pointing to the replacement.

It may be good practice to include something like
```
/**
 * @deprecated As of 1.0.11
*/
```

Reply:
Hi thanks for this suggestion. Last time when I checked this the docs said that there is no longer a need to write @deprecated tag for docs. It should be automatically generated thats why I left. But nonetheless I can add.

Reply:
I see; that should be fine. Please point me to where that is documented. I'm still figuring out our javadoc story.

Reply:
May be I am not finding the right link where I saw that anyways added that. Will let you know if I hit across that.